distribution/install.sh

```diff
@@ -209,26 +210,18 @@ add_to_path() {
 
 get_gnu_musl_glibc() {
   need_cmd ldd
-  need_cmd bc
   need_cmd awk
   # Detect both gnu and musl
-  # Also detect glibc versions older than 2.18 and return musl for these
-  # Required until we address https://github.com/timberio/vector/issues/5412.
   local _ldd_version
   local _glibc_version
-  _ldd_version=$(ldd --version)
-  if [[ $_ldd_version =~ "GNU" ]]; then
-    _glibc_version=$(echo "$_ldd_version" | awk '/ldd/{print $NF}')
-    if [ 1 -eq "$(echo "${_glibc_version} < 2.18" | bc)" ]; then
```

Comment:
I noticed we dropped this check. The associated issue is closed, but are we aware of the minimum libc version we currently require? It could be good to still include the check but update the minimum libc version.

Reply:
Ah, I'm not - @silversupreme had just noticed that the issue had been closed a while back. I'm not entirely sure how to verify this?

Reply:
This script seems to work pretty well: https://gist.github.com/fasterthanlime/17e002a8f5e0f189861c

The `x86_64-unknown-linux-gnu` build seems to require 2.14. We'd want to check `aarch64-unknown-linux-gnu` and `armv7-unknown-linux-gnueabihf` too. This could even be something we do in CI to enforce it.

Reply:
I'm fine with this being a follow-up but would suggest we replace the conservative check we have now until we can figure out the new min supported version.